### PR TITLE
Updating webpass doc

### PR DESCRIPTION
**Pull request by sreedhards**
Updating two pages in web pass

### CODE REVIEW

File: pages/web/web-paas/administration-users/guide.en-ca.md [hunk 1]
```diff
@@ -5,140 +5,163 @@ section: Administration
...
+>> 
+>> ```bash
+>> webpaas user:update user1@example.com -r production:v,development:c
+>> ```
+>> 
+>> After you change a user's role for an environment type, you must trigger a redeploy each environment to propagate access changes. You can redeploy using the CLI command `webpaas redeploy`.
+>> 
+>> ```     
+
+### Transfer project ownership
```
Comment (bs-ovh) [inline pages/web/web-paas/administration-users/guide.en-ca.md]: Could you please verify this part?

File: pages/web/web-paas/development-variables/guide.en-ca.md [hunk 2]
```diff
@@ -5,253 +5,268 @@ section: Development
...
 
-## Types
+If variables have the same names at different levels, the [variables are given precedence](#overrides) from bottom up. So Web PaaS-provided values override environment variables, which override project variables, which override application-provided variables.
 
-### Application-provided variables
+### Use application-provided variables
 
-Variables may be [set in code](../configuration-app/variables), using the `.platform.app.yaml` file.  These values will be the same across all environments and present in the Git repository, which makes them a poor fit for API keys and such.  This capability is mainly to define values that an application expects via an environment variable that should be consistent across all environments.  For example, some runtimes or frameworks use environment variables to configure their behavior, such as a `dev` vs `production` flag. You might want to set those as application-provided variables to ensure a consistent build across every environment. They may also be used to set [PHP configuration values](#php-specific-variables).
+Set variables [in code](../configuration/app/app-reference.md#variables) using the `.platform.app.yaml` file.
```
Comment (bs-ovh) [inline pages/web/web-paas/development-variables/guide.en-ca.md]: Link will not work

File: pages/web/web-paas/development-variables/guide.en-ca.md [hunk 3]
```diff
@@ -5,253 +5,268 @@ section: Development
...
+| ---------------- | ----- | ------- | ----------- |
+| PLATFORM_MODE    | No    | Yes     | `enterprise` in all production and staging Dedicated environments. Note that an Enterprise support plan doesn't always imply a Dedicated environment, but a Dedicated environment always implies an Enterprise support plan. |
+| PLATFORM_CLUSTER | No    | Yes     | The cluster ID. |
+| PLATFORM_PROJECT | No    | Yes     | The document root. Typically the same as your cluster name for the production environment, while staging environments have `_stg` or similar appended. |
 
-```bash
-echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
-```
+> [!primary]  
+> The `PLATFORM_CLUSTER`, and `PLATFORM_PROJECT` environment variables aren't yet available on [Dedicated Generation 3](../dedicated-gen-3-overview).
```
Comment (bs-ovh) [inline pages/web/web-paas/development-variables/guide.en-ca.md]: Link will not work
External service label, not Web PaaS

File: pages/web/web-paas/development-variables/guide.en-ca.md [hunk 4]
```diff
@@ -5,253 +5,268 @@ section: Development
...
+| PLATFORM_VARIABLES        | Some  | Some    | A base64-encoded JSON object with all project and environment variables that don't use a [prefix](#variable-prefixes). The keys are the variable names and the values the values. Availability during builds and at runtime depends on the specific variable settings. |
+| PLATFORM_PROJECT          | Yes   | Yes     | The project ID |
+| PLATFORM_TREE_ID          | Yes   | Yes     | The ID of the tree the application was built from, essentially the SHA hash of the tree in Git. Use when you need a unique ID for each build |
+| PLATFORM_PROJECT_ENTROPY  | Yes   | Yes     | A random, 56-character value created when the project is created and then stable throughout the project's life. Can be used for Drupal hash salts, Symfony secrets, and other similar values. |
+| PLATFORM_APP_DIR          | Yes   | Yes     | The absolute path to the application directory. |
+| PLATFORM_SOURCE_DIR       | No    | No      | Equivalent to `PLATFORM_APP_DIR` in the context of a running [source operation](../configuration/app/source-operations.md). The directory contains a writable copy of your repository that you can commit to during the operation. |
```
Comment (bs-ovh) [inline pages/web/web-paas/development-variables/guide.en-ca.md]: Link will not work

File: pages/web/web-paas/development-variables/guide.en-ca.md [hunk 5]
```diff
@@ -5,253 +5,268 @@ section: Development
...
+| PLATFORM_APPLICATION_NAME | Yes   | Yes     | The application name as set in the `.platform.app.yaml` file. |
+| PLATFORM_APPLICATION      | Yes   | Yes     | A base64-encoded JSON object that describes the application. It maps certain attributes from your `.platform.app.yaml` file, some with more structure. See [notes](#platform_application). |
+| PLATFORM_BRANCH           | No    | Yes     | The name of the Git branch. |
+| PLATFORM_DOCUMENT_ROOT    | No    | Yes     | The absolute path to the web document root, if applicable. |
+| PLATFORM_ENVIRONMENT      | No    | Yes     | The name of the Web PaaS environment. |
+| PLATFORM_ENVIRONMENT_TYPE | No    | Yes     | The type of the Web PaaS environment (development, staging or production). |
```
Comment (bs-ovh) [inline pages/web/web-paas/development-variables/guide.en-ca.md:156]: Not sure what this refers to, please verify.

File: pages/web/web-paas/development-variables/guide.en-ca.md [hunk 6]
```diff
@@ -5,253 +5,268 @@ section: Development
...
+| PLATFORM_SMTP_HOST        | No    | Yes     | The SMTP host to send email messages through. Is empty when mail is disabled for the current environment. |
+| PLATFORM_RELATIONSHIPS    | No    | Yes     | A base64-encoded JSON object of relationships. The keys are the relationship name and the values are arrays of relationship endpoint definitions. The exact format is defined for each [service](../configuration-services). |
+| PLATFORM_ROUTES           | No    | Yes     | A base64-encoded JSON object that describes the routes for the environment. It maps the content of the `.platform/routes.yaml` file. |
 
+Dedicated instances also have the following variables available:
 
-Since values can change over time, the best thing is to inspect the variable at runtime then use it to configure your application. For example:
+| Variable name    | Build | Runtime | Description |
+| ---------------- | ----- | ------- | ----------- |
+| PLATFORM_MODE    | No    | Yes     | `enterprise` in all production and staging Dedicated environments. Note that an Enterprise support plan doesn't always imply a Dedicated environment, but a Dedicated environment always implies an Enterprise support plan. |
```
Comment (bs-ovh) [inline pages/web/web-paas/development-variables/guide.en-ca.md]: External service labels / explanation

File: pages/web/web-paas/development-variables/guide.en-ca.md [hunk 7]
```diff
@@ -325,52 +340,251 @@ Check the individual documentation pages for accessing environment variables for
 >> 
 >> ```     
 
+### Accessing complex values
+
+Variable values can have nested structures. 
+The following example shows nested structures introduced in an [app configuration](../configuration/app/app-reference.md#variables):
```
Comment (bs-ovh) [inline pages/web/web-paas/development-variables/guide.en-ca.md]: Link will not work

**Review by bs-ovh — CHANGES_REQUESTED** [review]

File: pages/web/web-paas/development-variables/guide.en-ca.md (hunk 5 above)
Comment (sreedhards) [reply pages/web/web-paas/development-variables/guide.en-ca.md:156]: Looks ok, These variables are within Platform.sh

File: pages/web/web-paas/administration-users/guide.en-ca.md (hunk 1 above)
Comment (sreedhards) [reply pages/web/web-paas/administration-users/guide.en-ca.md]: removed the section

File: pages/web/web-paas/development-variables/guide.en-ca.md (hunk 6 above)
Comment (sreedhards) [reply pages/web/web-paas/development-variables/guide.en-ca.md]: Removed the section referenced to Dedicated

**Pull request merged**